pybamm/models/full_battery_models/lithium_ion/dfn.py

```diff
@@ -69,6 +69,27 @@ def set_active_material_submodel(self):
             self.submodels[
                 "positive active material"
             ] = pybamm.active_material.Constant(self.param, "Positive", self.options)
+        elif self.options["loss of active material"] == "both":
+            self.submodels[
+                "negative active material"
+            ] = pybamm.active_material.VaryingFull(self.param, "Negative", self.options)
+            self.submodels[
+                "positive active material"
+            ] = pybamm.active_material.VaryingFull(self.param, "Positive", self.options)
+        elif self.options["loss of active material"] == "anode":
+            self.submodels[
+                "negative active material"
+            ] = pybamm.active_material.VaryingFull(self.param, "Negative", self.options)
+            self.submodels[
+                "positive active material"
+            ] = pybamm.active_material.Constant(self.param, "Positive", self.options)
+        elif self.options["loss of active material"] == "cathode":
+            self.submodels[
+                "negative active material"
+            ] = pybamm.active_material.Constant(self.param, "Negative", self.options)
+            self.submodels[
+                "positive active material"
+            ] = pybamm.active_material.VaryingFull(self.param, "Positive", self.options)
```

Comment:
If we want to specify anode and cathode LAM independently, I think it would be better to have separate model.options for each one, i.e. "anode loss of active material" and "cathode loss of active material". Otherwise, if we add more LAM submodels, we need to take into account all of the combinations of negative and positive.

Reply:
Have no idea what will be the options for future LAM models. But I can discard those `cathode` and `anode` options and make it by modifing the LAM parameters. Does it make sense to you?

Reply:
That makes sense but then there will be equations like `deps/dt=0` which might unnecessarily slow down the solvers (I'm not sure how smart the solvers are to deal with easy equations like this). So it is good to be able to separately specify anode and cathode LAM submodels like you did.

Reply:
Thanks for yor reply. We can revisit this when we know what are the other LAM models exactly and how to make the options. I will keep the current structure and leave a placeholder for other LAM models.